### PR TITLE
Tear down Docker composition before bringing it up again

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request makes a change to tear down the [cisagov/orchestrator](https://github.com/cisagov/orchestrator) Docker composition before bringing it up again.

## 💭 Motivation and context ##

We have had an issue lately where the Docker composition does not start up correctly otherwise; this change will obviate said issue.

## 🧪 Testing ##

All automated tests pass.  I made this same change manually on the production system.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

### CODE REVIEW

**Comment by jsf9k** [comment]
:imp:

**Comment by jsf9k** [comment]
The GitHub Actions checks should pass once #661 is merged and this PR is rebased.

File: ansible/roles/orchestrator/tasks/main.yml [hunk 1]
```diff
@@ -64,7 +64,7 @@
     - name: Create a cron job for BOD 18-01 scanning
       ansible.builtin.cron:
         hour: '0'
-        job: cd /var/cyhy/orchestrator && docker compose up -d 2>&1 | /usr/bin/logger -t orchestrator
+        job: cd /var/cyhy/orchestrator && docker compose down && docker compose up --detach 2>&1 | /usr/bin/logger -t orchestrator
```
Comment (dav3r) [inline ansible/roles/orchestrator/tasks/main.yml]: As long as we are expanding from short-form to long-form option names:
```suggestion
        job: cd /var/cyhy/orchestrator && docker compose down && docker compose up --detach 2>&1 | /usr/bin/logger --tag orchestrator
```

**Review by dav3r — APPROVED** [review]

**Comment by jsf9k** [comment]
@mcdonnnj - Is there a reason why #661 can't be merged now?

**Review by dav3r — APPROVED** [review]
⬇️ ⬆️ 👍

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
This is a necessary evil right now but I would like an issue created to track hopefully unwinding this change once we sort out the underlying issue.

**Comment by jsf9k** [comment]
> This is a necessary evil right now but I would like an issue created to track hopefully unwinding this change once we sort out the underlying issue.

Please see issue #668 and commit cd78009.

**Review by mcdonnnj — APPROVED** [review]

**Pull request merged**